### ISSUE TITLE
What's the difference between 3D points generated from 2D keypoints and  generated from SMPL models?

### ISSUE THREAD

**Issue by aotaman** [google/aistplusplus_api#32]
hi, ruilong,
     Thanks for you great work, what's difference between keypoints3d_true and keypoints3d?  I use keypoints3d to generate smpl, but it didn't work. 

`
smpl_poses, smpl_scaling, smpl_trans = AISTDataset.load_motion(
                'motions', 'gBR_sBM_cAll_d04_mBR0_ch01')
  smpl = None
  smpl = SMPL(model_path='/home/public_data/sjc/smpl/models/', gender='MALE', batch_size=1)
  keypoints3d = smpl.forward(
      global_orient=torch.from_numpy(smpl_poses[:, 0:1]).float(),
      body_pose=torch.from_numpy(smpl_poses[:, 1:]).float(),
      transl=torch.from_numpy(smpl_trans/smpl_scaling).float(),
      ).joints.detach().numpy()[:, 0:24, :]

  keypoints3d_true = AISTDataset.load_keypoint3d(
      'keypoints3d', 'gBR_sBM_cAll_d04_mBR0_ch01', use_optim=False)

  smpl, loss = smpl_regressor.fit(keypoints3d, dtype='coco', verbose=True)
`

**Comment by aotaman** [comment]
I sloved it

**Comment by lvZic** [comment]
@aotaman I wonder how u solved it, thanks.
